src/Options/Definitions.js

```diff
@@ -137,7 +137,8 @@ module.exports.ParseServerOptions = {
   },
   "filesCacheControl": {
     "env": "PARSE_SERVER_FILES_CACHE_CONTROL",
-    "help": "Cache-Control header for files router"
+    "help": "Cache-Control header for files router",
+    "default": "public, max-age=86400"
```

Comment:
I don't think we want a default value here. Changing expected behavior by introducing caching would cause unexpected issues if someone needed to update their file within that 24 hr period. It would be best if it was unset as it was before.